assets/js/googlesitekit/datastore/site/reset.test.js

```diff
@@ -85,7 +85,7 @@ describe( 'core/site reset', () => {
 
 				// After a successful reset, `connection` should be `undefined` again.
```

Comment:
This comment is no longer accurate.

```suggestion
				// After a successful reset, `connection` state will be updated on the next page load.
```